backend/director/core/session.py

```
@@ -220,28 +220,31 @@ class OutputMessage(BaseMessage):
-        """Store the message in the database. for conversation history and publish the message to the socket."""
+        """Store in database and broadcast final result to session room."""
         self._publish()
 
     def _publish(self):
         try:
-            emit("chat", self.model_dump(), namespace="/chat")
+            emit("chat", self.model_dump(), 
+                 room=self.session_id, 
+                 namespace="/chat")
+            print(f"Emitted message to session room: {self.session_id}")
         except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)